grpc.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   s.platform      = Gem::Platform::RUBY
 
   s.add_dependency 'google-protobuf', '~> 3.7'
-  s.add_dependency 'googleapis-common-protos-types', '~> 1.0.0'
```

Comment:
Please also make this change in `templates/grpc.gemspec` (this file is generated form that one)

Reply:
Sure, done.